### DISCUSSION TITLE
ProjectV2 GraphQL API: programmatically editing options for a single select filed

### DISCUSSION THREAD

**Discussion by chicco785** [community/community#76762] [Apps, API and Webhooks]
### Select Topic Area

Product Feedback

### Body

Currently it is impossible to programmatically edit values for an existing single select field, if not using a UI. I hoped that this could be somehow possible using the create mutation, but it's impossible: the server return that the name is already taken.

Thant's quite a limit to be able to manage programmatically items in project, as we do, to create effective roadmaps.

**Comment by michaelconan** [comment]
Agreed, would love to see something like a `UpdateProjectV2Field` mutation for `ProjectV2Field`s, including `ProjectV2IterationField`, `ProjectV2SingleSelectField`, etc

I'd like to use it for a recurring GitHub action workflow to create new iteration (sprint) in my V2 project
